submissions/dummy_submission/the-nuclear-options.md

```diff
@@ -0,0 +1,146 @@
+## The Nuclear Option <author>Sir Fendi</author>
+
+When I came across a glowing green patch of uranium ore for the first time, I realized that Factorio had some juicy stuff waiting for me somewhere down the tech tree: Nuclear energy! While in today’s world the trials and triumphs of nuclear power are hotly debated, the Factorio community similarly has its ongoing discussions on whether it is worthwhile: Going nuclear is a couple steps up in complexity from the other power options, while also being completely optional on the path to launching a rocket. Hence many players tend to not bother with diving into it, or just find a good blueprint as a blackbox and call it a day. Others hear of the warnings of the nuclear reactor as a UPS killer, or even a literal killer when it explodes dramatically, and they decide to steer clear of it. Yes, there is a lot to look out for, but you can trust in Wube: Like many other carefully balanced mechanics of the game, nuclear power is reasonably easy to get into and you can quickly make something that just works. As for the more complicated and colorful part, I have a few guidelines to offer from my own fair share of tinkering and the findings of the glowing gems of our community.
+
+<!--Visual 1: A small nuclear setup with all the required parts, producing 40MW.-->
+
+### Demystifying Nuclear Power
+
+While a whole bunch of new machines may seem daunting, at its core, a vanilla nuclear plant setup is only a little different from a burner steam power setup: You have a fuel being burned, the resulting heat creating steam, and an engine making power from it. The extra complexity comes from three elements: First, the fuel used in reactors is unique and is made in multiple steps instead of using the mined resource directly. The official wiki has a [handy tutorial](https://wiki.factorio.com/Tutorial:Nuclear_power) that covers all the basics such as mining and centrifuging uranium ore, as well as setting up a basic single-reactor plant.
```

Comment:
```suggestion
While a whole bunch of new machines may seem daunting, at its core, a vanilla nuclear plant setup is only a little different from a burner steam power setup: You have a fuel being burned, the resulting heat creating steam, and an engine generating power from that steam. The extra complexity comes from three elements: First, the fuel used in reactors is unique and is made in multiple steps instead of using the mined resource directly. The official wiki has a [handy tutorial](https://wiki.factorio.com/Tutorial:Nuclear_power) that covers all the basics such as mining and centrifuging uranium ore, as well as setting up a basic single-reactor plant.
```